### PR TITLE
refactor vendor book add pagination

### PR DESCRIPTION
**Pull request by altyndev**

### CODE REVIEW

**Review by beksultancs — CHANGES_REQUESTED** [review]
I don't like this code

File: pom.xml [hunk 1]
```diff
@@ -127,7 +127,15 @@
 					</excludes>
 				</configuration>
 			</plugin>
-		</plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>14</source>
```
Comment (beksultancs) [inline pom.xml:134]: bul code deploydo ishtebeit* bul Verdi emne uchun koshtunuz

**Review by beksultancs — APPROVED** [review]
solve conflicts!

**Review by beksultancs — CHANGES_REQUESTED** [review]
need to solve problems

**Review by altyndev — COMMENTED** [review]
utACK

**Pull request merged**